packages/experimental/kai/src/containers/FrameRegistry/FrameRegistry.tsx

```diff
@@ -71,9 +69,9 @@ export const FrameRegistry: FC<{ slots?: FrameRegistrySlots }> = ({ slots = {} }
     switch (type) {
       case ExtensionType.FRAME: {
         const active = !activeFrames.find((frameId) => frameId === id);
-        setActiveFrame(id, active);
+        setActiveFrame(id, active); // TODO(burdon): Reconcile with navigation.
```

Comment:
I can try to reconcile the app navigation when I update shell to remove space selection from it